### ISSUE TITLE
Malformed version tag: v1.0-beta1

### ISSUE THREAD

**Issue by aofei** [juicedata/juicefs#1293]
Just a friendly reminder, a Go project should always follow [SemVer 2.0.0](https://semver.org) to name its version tags. Otherwise, it will not be able to be imported by other projects via Go modules. This project shouldn't be an exception, since some people are using it as a library. So please don't use `v1.0-beta1`, use `v1.0.0-beta1`; don't use `v1.0`, use `v1.0.0`.

**Comment by aofei** [comment]
```bash
$ go list -m -json github.com/juicedata/juicefs@v1.0-beta1
{
        "Path": "github.com/juicedata/juicefs",
        "Version": "v0.13.2-0.20220112051640-5d55cb4b00e7",
        "Time": "2022-01-12T05:16:40Z",
        "GoMod": "/home/aofei/go/pkg/mod/cache/download/github.com/juicedata/juicefs/@v/v0.13.2-0.20220112051640-5d55cb4b00e7.mod",
        "GoVersion": "1.16"
}
```

As you can see, `v1.0-beta1` is resolved to `v0.13.2-0.20220112051640-5d55cb4b00e7` because `v1.0-beta1` is not a valid version.

**Comment by davies** [comment]
@aofei thanks, create v1.0.0-beta1 to overwrite it: https://github.com/juicedata/juicefs/releases/tag/v1.0.0-beta1
